bin/serverless

```diff
@@ -2,26 +2,32 @@
 
 'use strict';
 
+const autocomplete = require('../lib/utils/autocomplete');
 const BbPromise = require('bluebird');
 const logError = require('../lib/classes/Error').logError;
 
-BbPromise.config({
-  longStackTraces: true,
-});
+if (process.argv[2] === 'completion') {
+  autocomplete();
+} else {
```

Comment:
purely style pref but could you do return; then not have to modify all the subsequent lines?